manifests/d/DominikReichl/KeePassClassic/1.43/DominikReichl.KeePassClassic.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.10.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.10.0.schema.json
+
+PackageIdentifier: DominikReichl.KeePassClassic
+PackageVersion: "1.43"
+PackageLocale: en-US
+Publisher: Dominik Reichl
+PackageName: KeePass 1.43

```

```suggestion
PackageName: KeePass
```
Versions should be left off of `PackageName`. Instead, use `AppsAndFeaturesEntries` with `DisplayName` if the version is important